subsys/debug/tracing/include/tracing_backend.h

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2019 Intel corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _TRACE_BACKEND_H
+#define _TRACE_BACKEND_H
+
+#include <sys/util.h>
+#include <tracing_packet.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Tracing backend
+ * @defgroup Tracing_backend Tracing backend
+ * @{
+ * @}
+ */
+
+#define TRACING_BACKEND_UART_NAME "tracing_backend_uart"
```

Comment:
This header is generic, so why have it tainted by UART stuff?

Reply:
I will use the name of backend to get the working backend instance, I think this is the most suitable file where we place them.